content/trust-management/trust-authority.md

```diff
@@ -115,14 +116,41 @@ Where:
+| `signature` | `string`          |  x   | Signature generated from a `signedNodeObject`, containing the node's server name, using one of the trust authority's public keys and the algorithm provided under `sig_algo`, and following the instructions described [here](/information-distribution/signature/#signing-json-data). |
+| `reason`    | `localisedString` |      | Reason given by the TA explaining why they trust this node.                                                                                                                                                                                                                            |
+
+* `signedNodeObject` is a map using the following structure:
+
+| Parameter    | Type              | Req. | Description                                                                                                                        |
+|:-------------|:------------------|:----:|:-----------------------------------------------------------------------------------------------------------------------------------|
+| `serverName` | `string`          |  x   | The node's server name.                                                                                                            |
+| `reason`     | `localisedString` |      | Reason given by the TA explaining why they trust this node. This value **must** match the reason provided alongside the signature. |
+
+* `blacklistEntries` is a map using the following structure:
+
+| Parameter  | Type                  | Req. | Description                                                                        |
+|:-----------|:----------------------|:-----|:-----------------------------------------------------------------------------------|
+| `entities` | `blacklistedEntities` |      | Entities (sources and other trust authorities) blacklisted by the trust authority. |
+| `nodes`    | `blacklistedNodes`    |      | Nodes blacklisted by the trust authority.                                          |
+
+* `blacklistedEntities` is a map associating a Matrix user ID to a
+  `blacklistEntry`.
+
+* `blacklistedNodes` is a map associating a node's server name (as a string) to
+  a `blacklistEntry`.
+
+* `blacklistEntry` is a map using the following structure:
+
+| Parameter     | Type              | Req. | Description                                                                                                                     |
+|:--------------|:------------------|:----:|:--------------------------------------------------------------------------------------------------------------------------------|
+| `reason_code` | `string`          |  x   | One of the reason codes defined [above](#blacklist-reason-codes).                                                               |
+| `after`       | `string`          |      | ID of the latest trustworthy event sent by the entry's target. Empty string or omit key if none (e.g. with `B_MISINFORMATION`). |
```

Comment:
The after key is allowed to be present for both blacklisted entities and nodes. I don't know If it's intended (since the node blacklist is effectively a blacklist against all its users) or not.

Reply:
Yes it is intended, and I don't understand your point on why it seems weird to do so.